### PR TITLE
build everything on github actions

### PR DESCRIPTION
**Pull request by robstoll**
Since travis starts to be unreliable with its credit system.

______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: .github/workflows/bbc-ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: BBC Ubuntu
```
Comment (jGleitz) [inline .github/workflows/bbc-ubuntu.yml:1]:
```suggestion
name: Backwards-Backwards Compatibility Ubuntu
```

When I started developing on Atrium, it took me a long time to figure out what a British TV network has to do with my Pull Request :wink: . I think giving this a proper name helps contributors.

File: .github/workflows/bc-ubuntu.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: BC Ubuntu
```
Comment (jGleitz) [inline .github/workflows/bc-ubuntu.yml:1]:
```suggestion
name: Backwards Compatibility Ubuntu
```

File: .github/workflows/bbc-ubuntu.yml (hunk 1 above)
Comment (robstoll) [reply .github/workflows/bbc-ubuntu.yml:1]: Fair enough :)

Comment (robstoll) [reply .github/workflows/bbc-ubuntu.yml:1]: btw. it means binary backward compatiblity (fixed it directly in master)

**Pull request merged**